src/modules/core/components/MiniSpinnerLoaderWrapper/MiniSpinnerLoaderWrapper.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactNode, useEffect, useState } from 'react';
+
+import { MiniSpinnerLoader } from '~core/Preloaders';
+
+interface Props {
+  children: ReactNode;
+  milliseconds: number;
+}
+
+const MiniSpinnerLoaderWrapper = ({ children, milliseconds }: Props) => {
```

Comment:
I think you could choose a better name for the component. Being specific about what it has/does more than `MiniSpinnerLoader` (not more general).

Also `milliseconds` is a wasted opportunity of a variable name. It doesnt tell me whats it being used for, or give a context. 
A unit type as a name could be confusing.

Reply:
Good points. I will change `milliseconds` to something like `loadingDuration`. Any suggestion for a better component name?

Reply:
> Good points. I will change milliseconds to something like loadingDuration. Any suggestion for a better component name?

I'm trying to improve my own naming convention thought process.  I need to daydream for a while...... My default is usually to be very literal. 

Probably `MiniSpinnerLoader` isnt a great name (Mini) to start with, as the appearance can easily be changed with a prop. But now you have to build upon it!!! (or start changing lots of things)

Do we even need another `Mini-SpinnerLoader-Wrapper` component? We have lots and they are all very similar. 

I think that having a timeout is a useful feature in a spinner. 
So you should not make another component but instead, add the timeout to the 'base' component SpinnerLoader. Add the `loadingDuration` prop (default=0) then everyone can make use of it, or not.

..... and for the extra work, you get to avoid the problem of deciding on an expressive name ;)